pycbc/workflow/coincidence.py

```diff
@@ -655,3 +688,31 @@ def setup_multiifo_interval_coinc(workflow, hdfbank, trig_files, stat_files,
 
     logging.info('...leaving coincidence ')
     return statmap_files
+
+def get_ordered_ifo_list(ifocomb, ifo_ids, insps):
+    # combination_prec stores precedence info for the detectors in the combination
```

Comment:
please add a docstring that describes what this function does

Reply:
done